docs/source/reference/yaml-spec.rst

```diff
@@ -143,7 +143,7 @@ Available fields:
       # Uses SkyPilot Storage to create a S3 bucket named sky-dataset, uploads the
       # contents of /local/path/datasets to the bucket, and marks the bucket
       # as persistent (it will not be deleted after the completion of this task).
-      # Symlink contents are copied over.
+      # Symlink contents are directly copied, but not their target data.
```

Comment:
Thanks for catching this - I think it should be this, since we do not copy over symlinks or their data when uploaded using Sky Storage. 

```suggestion
      # Symlinks and their contents are NOT copied.
```